eclipsePlugin-junit/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "$rootDir/gradle/checkstyle.gradle"
+
+dependencies {
+  compile project(':eclipsePlugin')
+  testCompile 'junit:junit:4.12'
+  testCompile 'org.mockito:mockito-core:2.7.22'
+}
```

Comment:
My point is: ideally
1. we can run Eclipse tests automatically
2. we have one project for Eclipse tests (otherwise questions arise which test to put where and this is not obvious).

Unfortunately point 1. is not true and so you have started a new project which allows some subset of tests to run, and this is against point 2. . 

My previous comment was: if you are going to add a new project for Eclipse tests because we can't run the tests in the old project, please add readme files in each of them explaining the situation, and a TODO to merge the new project with the old one as soon as we can run all tests automatically.